frontend/src/components/FormItem.js

```diff
@@ -37,14 +37,19 @@ function LabelWrapper({ label, children, className }) {
 LabelWrapper.propTypes = labelPropTypes;
 
 function FormItem({
-  label, children, required, name, fieldSetWrapper, className,
+  label, children, required, name, fieldSetWrapper, className, requiredText,
```

Comment:
Do we need to give the option to change the `requiredText`? or can we just hardcode it to "*"?

Reply:
I will ask Kelly if they want this change to reflect over the entire app.

Reply:
Confirmed they want it across the app. Ill remove this prop.